### ISSUE TITLE
[Textos Blog]

### ISSUE THREAD

**Issue by 1leonardovieira** [SouJunior/products#661]

**Comment by 1leonardovieira** [comment]
![Image](https://github.com/user-attachments/assets/72f6d96b-51d3-4bf9-9447-0b6a3a49f6e1)

@Vanessavpcs  - Olhando o site, seria interessante um botão para direcionar para o blogsouJunior

**Comment by Vanessavpcs** [comment]
@1leonardovieira concordo! Pode pedir para a Livia ou a Belit levar essa ideia para as APMs do Site (Thaymara e Sabrina).
